src/Amazon.Common.DotNetCli.Tools/Utilities.cs

```diff
@@ -694,5 +694,33 @@ public static ExecuteShellCommandResult ExecuteShellCommand(string workingDirect
                 return new ExecuteShellCommandResult(proc.ExitCode, capturedOutput.ToString());
             }            
         }
+
+        public static string ReadSecretFromConsole()
```

Comment:
Is the current behavior of accepting ESC and TAB characters valid? Also, is setting the key to empty the desired effect for just pressing enter or should it check for this case?

Reply:
Updated code to skip the initial ASCII control characters like ESC and TAB